pkgs/tools/video/harvid/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, ffmpeg, libjpeg, libpng, pkg-config }:
+stdenv.mkDerivation {
+  pname = "harvid";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = "harvid";
+    rev = "d71921ba52efd1362973df850213eebe97a7ecc6";
+    sha256 = "0l1plfsfh2ixhlzg3hqqvjj42z7g422718a9kgbh7b4p882n71x7";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ ffmpeg libjpeg libpng ];
+
+  installPhase = ''
+    runHook preInstall
+    make DESTDIR="$out" libdir="/lib" install
```

Comment:
Please convert this into makeFlags and postInstall.

Reply:
Thanks for the tip!